src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -44,30 +44,11 @@ public void testDoGet() throws IOException, ServletException {
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
   }
 
-  public void testDoPost_ExistingUser() throws IOException, ServletException {
```

Comment:
Why did we decide to delete this test? It is okay to have multiple test cases for doPost, in fact it is great to have multiple test cases if there are different conditions you want to test.

Reply:
I could not get the test to work for some reason, could we go over this more in our one on one?